datajunction-server/alembic/versions/2023_07_07_2006-4e1ff36c27c6_initial_migration.py

```diff
@@ -176,6 +176,23 @@ def upgrade():
             name=op.f("pk_catalogengines"),
         ),
     )
+    op.create_table(
```

Comment:
@agorajek This actually breaks the existing db migration setup, unless we squeeze all of the migrations into this initial commit